src/main/java/org/testng/internal/Version.java

```diff
@@ -2,7 +2,7 @@
 
 public class Version {
 
-    public static final String VERSION = "7.0.1";
+    public static final String VERSION = "7.1.1-SNAPSHOT";
```

Comment:
Please exclude this file from the changeset. This is a generated class and should not be added to the changeset.